MAINTAINERS.yml

```diff
@@ -846,6 +846,18 @@ MCU Manager:
     labels:
         - "area: mcumgr"
 
+OSDP:
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
```

Comment:
```suggestion
    collaborators:
```

Reply:
Is someone else planning on taking up the maintership of this code?

Reply:
@cbsiddharth maintainers have to be approved by TSC, and this is not the case yet here.

Reply:
Okay, changed to `collaborators`.

Reply:
Some more context [here](https://docs.zephyrproject.org/latest/contribute/project_roles.html)